### DISCUSSION TITLE
IDU video demos added to the module

### DISCUSSION THREAD

**Discussion by chjch** [SERVIR-WA/GALUP#408] [Announcements]
Dear all,

We just added three video demos to the training material. They can be found under Section 4 after each workflow diagram. https://github.com/SERVIR-WA/GALUP/blob/master/training/3_idu/module.md#41-developing-urban-clusters

I believe these videos will be tremendously helpful in clarifying how to work with specific tools in the process. Please check the videos and let us know if you have any more questions.

Best,

GALUP Team

**Comment by Achidago** [comment]
![Screenshot (110)](https://user-images.githubusercontent.com/85199074/164715592-0f43650b-7a0d-4516-8f08-33c37cd62492.png)
![Screenshot (111)](https://user-images.githubusercontent.com/85199074/164715603-c7c626d4-4490-42ac-a659-690254b7894e.png)
![Screenshot (112)](https://user-images.githubusercontent.com/85199074/164715613-670b4272-d059-4728-90be-bd784dcc0bec.png)
![Screenshot (113)](https://user-images.githubusercontent.com/85199074/164715622-ffd0783f-7a48-4a74-9f2b-d4b3fde8bb30.png)
This is the output i'm getting after cobining the clusters. Kindly check and advice

**Reply by chjch** [reply]
Hi Silas @Achidago, I want to clarify that parameters for individual tools used in the demo and the training module are specific to THLD district. They should be somewhat adjusted when you apply the workflow to AAC district. For example, you should check the result of DBSCAN and see if they are reasonable. If not, some modifications need to be made to the parameters, i.e., smallest size of cluster and maximum distance. I see that you have five clusters detected. But you may want to check whether those five clusters are reasonably identified. 

For the weights used in the raster calculator, they are based on the relative magnitude of each cluster, which can be determined by the “CLUSTER_SIZE” of the output of DBSCAN clustering. For example, let’s say there are three clusters identified with size 25, 35, and 40, respectively. Then, the weights, correspondingly, should be 0.25, 0.35, and 0.4 for each cluster. These weights should be used along with the corresponding proximity layer in the raster calculator to get the merged proximity map. 

Please let me know my description is clear. If you have any further questions, please let me know.

**Comment by Achidago** [comment]
Thanks for this. I will have a second look at it.

**Comment by Achidago** [comment]
Combined Cluster Raster
![Screenshot (115)](https://user-images.githubusercontent.com/85199074/165045479-555fcac5-8113-4ad7-a0bd-f8d7cfb8711b.png)

Reclassified Cluster Raster
![Screenshot (114)](https://user-images.githubusercontent.com/85199074/165045513-b012e11f-025c-4d58-986a-76198d663616.png)

If this is right, how do i extract the rural and urban rasters from the 
reclassified combined raster?
